Sources/RingPublishingTracking/Private/Decorator/Generic/UserDataDecorator.swift

```diff
@@ -16,10 +16,10 @@ final class UserDataDecorator: Decorator {
         return encoder
     }()
 
-    private var data: UserData? = UserData(sso: nil, id: nil)
+    private var data: UserData = UserData()
```

Comment:
tu byś się upewnił @bbijoch że to nie poleci z niczym jak nic nie jest ustawione

Reply:
@adamszeremeta Przy produkowaniu RDLU są `guard'y` na wartości i będzie OK. Nic nie poleci